tests/regress/sql/test_plpgsql.sql

```diff
@@ -0,0 +1,7 @@
+CREATE SCHEMA s1;
+
+DO $plpgsql$BEGIN
+    PERFORM diskquota.set_schema_quota('s1', '1 MB');
+END;$plpgsql$;
```

Comment:
It is redundant to write `plpgsql` and semicolon after `END`. See example in PostgreSQL documentation https://postgrespro.ru/docs/postgrespro/9.5/sql-do?lang=en
```suggestion
DO $$BEGIN
    PERFORM diskquota.set_schema_quota('s1', '1 MB');
END$$;
```

Reply:
done